sunpy/timeseries/sources/eve.py

```diff
@@ -57,11 +57,14 @@ class ESPTimeSeries(GenericTimeSeries):
 
     _source = 'esp'
 
-    @peek_show
-    def peek(self, title='EVE/ESP Level1', **kwargs):
+    def plot(self, axes=None, title="EVE/ESP Level1", **kwargs):
         """
+        Plots the EVE ESP Leve1 timeseries data.
+
         Parameters
         ----------
+        axes : `matplotlib.axes.Axes`
+            The axes on which to plot the TimeSeries. Defaults to current axes.
```

Comment:
Now we need a return section in the doc string.

Reply:
Ah, will do that.